test/cppsuite/configs/hs_cleanup_stress.txt

```diff
@@ -32,9 +32,8 @@ runtime_monitor=
     ),
     stat_cache_size=
     (
-        # FIXME-WT-7848: Enable cache usage check once cache exceed failure is fixed.
         max=110,
```

Comment:
Should we limit to `100` now?

Reply:
Hmm good question, I am thinking that we maintain 110 for now and re-evaluate this when we understand the problem in WT-8437 more.